Makefile

```diff
@@ -0,0 +1,52 @@
+# Shell to use with Make
+SHELL := /bin/bash
+
+# Set important Paths
+PROJECT := btrdbextras
+LOCALPATH := $(CURDIR)/$(PROJECT)
+
+# Export targets not associated with files
+.PHONY: test grpc
+
+# Clean build files
+clean:
+	find . -name "*.pyc" -print0 | xargs -0 rm -rf
+	find . -name "__pycache__" -print0 | xargs -0 rm -rf
+	find . -name ".DS_Store" -print0 | xargs -0 rm -rf
+	-rm -rf docs/build
+	-rm -rf htmlcov
+	-rm -rf .pytest_cache
+	-rm -rf .coverage
+	-rm -rf build
+	-rm -rf dist
+	-rm -rf $(PROJECT).egg-info
+	-rm -rf .eggs
+	-rm -rf site
+	-rm -rf docs/_build
+	-rm -rf platform-builds meta.yaml
+
+# Generate new grpc code
+grpc:
+	@echo Generating files:
+	python -m grpc_tools.protoc -I eventproc/protobuff --python_out=eventproc/protobuff --grpc_python_out=eventproc/protobuff eventproc/protobuff/api.proto
```

Comment:
Minor nit: usually it's just one f.

Reply:
Ugh, you're absolutely right.  I'm gonna leave as is but change in a future PR if you dont mind.